pages/migration/wsc-53/migration_wsc-53_php.md

```diff
@@ -0,0 +1,19 @@
+---
+title: Migrating from WSC 5.3 - PHP
+sidebar: sidebar
+permalink: migration_wsc-53_php.html
+folder: migration/wsc-53
+---
+
+## Minimum requirements
+
+The minimum requirements have been increased to the following:
+
+- **PHP:** 7.2
+- **MySQL:** 5.7 / 8.0
+- **MariaDB:** 10.1
```

Comment:
Shouldn't there also be the patch version as 7.2.0 is a PHP 7.2, but we require 7.2.24?

Reply:
This is the developer documentation, not the end user documentation. No functional changes are expected between 7.2.0 and 7.2.24. I'm intentionally only putting the branch here for clarity.

Reply:
I still think that we should specify system requirements as precisely as possible, in particular as adding just another digit does not really change readability.

Reply:
Having another place where we document these requirements isn't a bad idea, makes it easier to find the exact requirements.